examples/kong/start

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
should this have an `examples/kong/README.md` file too? what are you trying to demonstrate with this example? I think it could be clearer here

Reply:
In contrast to the other two examples of Amazon API Gateway and Apigee, Kong doesn't let you import an API spec and run with it. I figured it would be worth it to show an example of converting the spec to something it accepts, and setting it up in its most basic form.